### DISCUSSION TITLE
在Unraid系统下通过本机存储模式挂载本地后新建的文件和文件夹会权限不足

### DISCUSSION THREAD

**Discussion by AdrianLogo** [AlistGo/alist#4472] [Q&A]
![image](https://github.com/alist-org/alist/assets/60739181/49c8933b-1c72-4abf-8aef-1068292e032d)
![image](https://github.com/alist-org/alist/assets/60739181/aca82165-bf21-46db-b3fa-04358fbc5692)

这是我设置的创建文件夹权限777和0777都试过了

![image](https://github.com/alist-org/alist/assets/60739181/f4ef5ec5-d6ae-4277-a9e1-fff34ecf68e0)
![image](https://github.com/alist-org/alist/assets/60739181/5f773778-4eb2-4c6e-b827-06851b5bcbbc)

这是实际创建出来的文件夹和文件，权限只有755
这导致除了管理员外其他用smb的用户会权限不足，无法对文件夹和文件进行操作，目前我只能通过用smb模式挂载后进行操作，这样权限就正常了，求大佬能解决一下这个问题

**Comment by Jiao-Xia-Ke** [comment]
我也遇到了,而且还不能设置用户和组

**Reply by nickweiminhan** [reply]
我也遇到了
